### ISSUE TITLE
Val dataset's usage?

### ISSUE THREAD

**Issue by codenewww** [Junjue-Wang/EarthVQA#14]
Thank you for your code! I would like to know if, in your implementation, the validation dataset was not used at all? Was the validation also conducted on the training dataset?

**Comment by Junjue-Wang** [comment]
I use the validation set to try my hyperparameters.

**Comment by codenewww** [comment]
Thank you for your response. Do you mean that the validation set was used to try hyperparameters during the training of both the segmentation network and the SOBA network, or was it used in only one of these stages? Thank you!

**Comment by Junjue-Wang** [comment]
Both of seg and VQA are involved

**Comment by codenewww** [comment]
OK. Thanks again!
